### PR TITLE
Added region and zone information on nodes' details and list

### LINKED ISSUE #1837: Introduce region, zone for Node details and provider information.
**Description**

Provide extra information related to the zone that the node runs. This information will be displayed at list level and detail level
```
       "topology.kubernetes.io/region": "europe-west4",
         "topology.kubernetes.io/zone": "europe-west4-b",
```
Add information about the provider from the [Gardener CM](https://github.com/gardener/gardener/blob/master/docs/usage/shoot_info_configmap.md)

Acceptance criteria:
- Region and zone information is displayed on Node's details and list.
- Provider name is displayed in the Cluster Overview header. It should be checked whether the Feature flag is enabled and the user has access to the `kube-system` Namespace where the Gardener CM is located.

### PR DESCRIPTION
**Pull request by Lyczeq**
**Description**

Changes proposed in this pull request:

- Added region and zone information on nodes' details and list
- Adjusted translations
- Adjusted kyma version tests and added a test for a gardener provider
- Changed the way Kyma version is displayed. When Kyma is not installed or there is an error, the information isn't displayed.
- Added conditional (via feature flag) displaying the Gardener provider

**Related issue(s)**
Closes #1837

### CODE REVIEW

**Comment by Lyczeq** [comment]
/test pre-busola-smoke-test-docker


<br class="Apple-interchange-newline">

**Review by akucharska — DISMISSED** [review]

File: docs/features.md [hunk 1]
```diff
@@ -140,6 +140,44 @@ Note that some features must be run before the application starts the bootstrap
   }
   ```
 
+* **GZIP** – is used to indicate whether the response from the backend server should be compressed or not.
```
Comment (grego952) [inline docs/features.md:143]:
```suggestion
* **GZIP** – is used to indicate whether a response from the backend server should be compressed or not.
```

File: docs/features.md [hunk 2]
```diff
@@ -140,6 +140,44 @@ Note that some features must be run before the application starts the bootstrap
...
+
+Backend feature. Cannot be modified at the cluster's ConfigMap level.
```
Comment (grego952) [inline docs/features.md:145]:
```suggestion
> **NOTE:** It's a backend feature, so it cannot be modified at the cluster's ConfigMap level.
```

File: docs/features.md [hunk 3]
```diff
@@ -365,7 +365,15 @@ The **match** keys and **messageSrc** must use the format described in the [`jso
   },
   ```
 
-- **SHOW_KYMA_VERSION** – determines if the Kyma version should be visible on the Cluster Details page. The displayed version is the value of the `reconciler.kyma-project.io/origin-version` label in the `kyma-system` Namespace. If the value of the label is missing or there is no `kyma-system` Namespace, the `Unknown` version will be displayed.
+- **SHOW_GARDENER_METADATA** - determines if the metadata given from the Gardener should be displayed. The displayed information is the value from the `shoot-info` ConfigMap based in the `kube-system` namespace. If the ConfigMap doesn't exist, the information won't be displayed.
```
Comment (grego952) [inline docs/features.md:368]:
```suggestion
- **SHOW_GARDENER_METADATA** - determines if the metadata taken from Gardener should be displayed. The displayed information is the value from the `shoot-info` ConfigMap based on the `kube-system` Namespace. If the ConfigMap doesn't exist, the information is not displayed.
```

File: docs/features.md [hunk 4]
```diff
@@ -365,7 +365,15 @@ The **match** keys and **messageSrc** must use the format described in the [`jso
...
+
+  ```json
+  "SHOW_GARDENER_METADATA": {
+    "isEnabled": true
+  },
+  ```
+
+- **SHOW_KYMA_VERSION** – determines if the Kyma version should be visible on the Cluster Details page. The displayed version is the value of the `reconciler.kyma-project.io/origin-version` label in the `kyma-system` Namespace. If the value of the label is missing or there is no `kyma-system` Namespace, the information won't be displayed.
```
Comment (grego952) [inline docs/features.md:376]:
```suggestion
- **SHOW_KYMA_VERSION** – determines if the Kyma version should be visible on the **Cluster Details** page. The displayed version is the value of the `reconciler.kyma-project.io/origin-version` label in the `kyma-system` Namespace. If the value of the label is missing or there is no `kyma-system` Namespace, the information is not displayed.
```

**Review by grego952 — APPROVED** [review]

**Review by akucharska — APPROVED** [review]

**Pull request merged**